src/backend/distributed/sql/citus--11.0-4--11.1-1.sql

```diff
@@ -167,3 +167,6 @@ GRANT SELECT ON pg_catalog.pg_dist_background_task_depend TO PUBLIC;
 
 #include "udfs/citus_job_wait/11.1-1.sql"
 #include "udfs/citus_job_cancel/11.1-1.sql"
+#include "udfs/citus_rebalance_start/11.1-1.sql"
+#include "udfs/citus_rebalance_stop/11.1-1.sql"
+#include "udfs/citus_rebalance_wait/11.1-1.sql"
```

Comment:
Maybe we can include `citus_job_list`/`citus_rebalance_list` as a `select * from pg_catalog.pg_dist_background_job` so that we don't expose the internal storage if someone needs to find the job id/see what is present there.

Reply:
We quite often allow users to query catalog tables directly. Adding these functions don't add much imho. Especially since the jobs table does not contain much information on task status.

I think to do this correctly we should have a detailed idea on what we want to show specifically for rebalances, which we postponed till later.